doc/containers.md

```diff
@@ -138,11 +138,16 @@ LXD supports the following device types:
 
 ID (database)   | Name          | Description
 :--             | :--           | :--
-0               | none          | Inheritance blocker
-1               | nic           | Network interface
-2               | disk          | Mountpoint inside the container
-3               | unix-char     | Unix character device
-4               | unix-block    | Unix block device
+0               | [none](#type-none)  | Inheritance blocker
+1               | [nic](#type-nic)        | Network interface
+2               | [disk](#type-disk)          | Mountpoint inside the container
+3               | [unix-char](#type-unix-char)     | Unix character device
+4               | [unix-block](#type-unix-block)    | Unix block device
+?               | [unix-usb](#type-usb)    |
+?               | [unix-gpu](#type-gpu)    |
+
+
+
```

Comment:
Can you fix the alignment of the column separators too? Basically just pushing the description column a bit to the right so that they all align again.